runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -38,6 +38,14 @@
 //! costs and don't want to just run everything in order (as that would be to
 //! slow), we have a very simple manual caching infrastructure in place.
 //!
+//! To run estimations on a non-empty DB with standardised content, we first
+//! dump all records to a `StateDump` written to a file. Then for each
+//! iteration of a an estimation, we first load the records from this dump into
+//! a fresh database. Afterwards, it is crucial to run compaction on RocksDB
+//! before starting measrments. Otherwise the SST file layout can be very
+//! ineifficient, as there wass no time to restructure them. Also, compaction
```

Comment:
```suggestion
//! inefficient, as there was no time to restructure them. Also, compaction
```

Reply:
And maybe make a higher level comment a-la "we assume that production database is reasonably compacted".

Reply:
done